plugin/completion/lib_complete.py

```diff
@@ -82,6 +83,11 @@ def __init__(self, clang_binary):
             log.debug(
                 " using bundled cindex: %s", cindex_module_name)
             cindex = importlib.import_module(cindex_module_name)
+
+            Completer.ignore_list.append(cindex.CursorKind.DESTRUCTOR)
```

Comment:
Is there any reason to use 3 appends versus just specifying a list `[kind_1, kind_2, kind_3]`?